LDAR-Sim_inputs.md

```diff
@@ -26,6 +26,263 @@ To support wider use of LDAR-Sim, the University of Calgary and Highwood Emissio
+
+To ensure reproducibility, the full set of parameters are written in the output directory so the model run can be reproduced or interrogated.
+
+Keep in mind that the **order matters** - if you change around the order that parameter files are read in, the parameters in the end will be different.
+
+The example above focused on a testing run configuration, but the functionality is designed to be used to enable modularization. For example, parameter files can be modularized into categories like:
+
+- specific asset bases and infrastructure types
+- specific program definitions
+- specific method definitions
+- specific run configurations (e.g., as example above)
+- specific scheduling configurations
+- specific computer systems
+
+For example, running an existing program with an existing collection of methods in a new jurisdiction is as simple as just revising the asset parameter file to the new set of sites, leaving the other parameter files untouched.
+
+While global parameters are straightforward to specify this way, and the above example shows how to do this - a few extra parameters are required to directly specify programs or methods, which are at different levels in the hierarchy.
+
+## LDAR-Sim Parameter Hierarchy
+
+LDAR-Sim uses a 3 level hierarchy of simulations, programs, and methods. To tell LDAR-Sim what level in the hierarchy your parameter file is destined for, you should specify a `parameter_level` parameter that will specify what level your parameter file is aimed at - otherwise LDAR-Sim will interpret it as global.
+
+The `parameter_level` parameter can be one of three values:
+
+- `global`: parameters are aimed at the global level.
+- `program`: parameters are used as a program definition.
+- `method`: parameters are used as a method definition.
+
+While this is relative intuitive, there are special considerations for methods:
+
+- First, methods have a `module` key, that relate to specific method modules. For example, an OGI method is simulated using the OGI module. Users can build custom types or extend existing types, but some `module` is necessary to ensure LDAR-Sim knows what code to run. The full list of available modules is provided in parameter documentation below.
+
+- Second, methods have labels. This is necessary as there can be many different methods that are quite different, but have the same `module`. A good example is different OGI companies. You can run a simulation with multiple OGI companies, each specified as a unique method with unique agents, but both of the same type `OGI`. This is helpful to represent different work practices, different collection of parameters, or different approaches with the same technology.
+
```

Comment:
Can we include a second example (mostly just to see if I'm getting this right):

"If you are modeling a hypothetical aerial monitoring company called 'Happy Leaks' it will have a module set to 'aircraft' and a label set to 'Happy Leaks'."